src/tests/Microsoft.PowerFx.Connectors.Tests/Microsoft.PowerFx.Connectors.Tests.csproj

```diff
@@ -8,10 +8,13 @@
 		<OutputType>Library</OutputType>		
 	</PropertyGroup>
 	<ItemGroup>
-	  <None Remove="Swagger\MSNWeather.json" />
-	  <None Remove="Swagger\MSNWeather_Response.json" />
-	  <None Remove="Swagger\TestOpenAPI.json" />
-	  <None Remove="Swagger\TestOpenAPI2.json" />
+    <None Remove="Responses\AzureBlobStorage_Response.json" />
+    <None Remove="Swagger\AzureBlobStorage.json" />
+    <None Remove="Swagger\MSNWeather.json" />
+    <None Remove="Swagger\MSNWeather_Response.json" />
+    <None Remove="Swagger\TestOpenAPI.json" />
+    <None Remove="Swagger\TestOpenAPI2.json" />
+    <None Remove="Swagger\TestOpenAPI3.json" />
 	</ItemGroup>
```

Comment:
Do lines 34..40 cover this, can we remove this whole section?

Reply:
VS...
Fixed